src/paper.rs

```diff
@@ -29,14 +30,37 @@ impl PaperList {
         });
         table.set_header(header);
 
+        // Get read and current colors from the config.
+        let mut read_color: Result<Color, ()> = Err(());
+        let mut current_color: Result<Color, ()> = Err(());
+
+        if let Some(label_colors) = &config.output.label_colors {
+            if let Some(color) = label_colors.get("read") {
+                read_color = Color::try_from(color.as_str());
+            }
+
+            if let Some(color) = label_colors.get("current") {
+                current_color = Color::try_from(color.as_str());
+            }
+        }
+
```

Comment:
If I understood correctly, this will only colorize labels `read` and `current`. If the user adds, for example, `good = "Red"` to `config.toml`, it won't do anything, which makes it meaningless to allow generic configuration of `label_colors`. The goal of this feature as I understood was to allow users to (1) use any label they like (without being forced to only using colors for marking reading progress), and (2) arbitrarily associate a subset of their labels with a color they would like.

In order to implement (1) and (2), for each `Paper` in `PaperList`, we need to check whether that `Paper` has a label whose color has been configured in `config.output.label_colors`. That is essentially checking whether the set intersection operation between the keys of `config.output.label_colors` and `paper.labels` is non-empty. If there is more than one label in the intersection set, that means label colors collided, and we would have to make a decision -- whether to error out, or to still select one color. My immediate preference would be to choose the one that is later in the label list, because that would be the most recent label given to the paper by the user.